src/Constants.ts

```diff
@@ -5,7 +5,7 @@ import MySQLIntegration from './plugins/integrations/MySQLIntegration';
 import RedisIntegration from './plugins/integrations/RedisIntegration';
 import MongoDBIntegration from './plugins/integrations/MongoDBIntegration';
 import IORedisIntegration from './plugins/integrations/IORedisIntegration';
-import { AWSIntegration } from './plugins/integrations/AWSIntegration';
+import {AWSIntegration, AWSSESIntegration} from './plugins/integrations/AWSIntegration';
```

Comment:
`AWSSESIntegration` import should not be needed as no need to put `AWSSESIntegration` into `INTEGRATIONS` because the main `AWSIntegration` is already there